torchx/mix.exs

```diff
@@ -19,7 +19,14 @@ defmodule Torchx.MixProject do
       compilers: compilers() ++ Mix.compilers(),
       elixirc_paths: elixirc_paths(Mix.env()),
       aliases: aliases(),
-      make_env: %{"LIBTORCH_DIR" => System.get_env("LIBTORCH_DIR", @libtorch_cache)}
+      make_env: %{
+        "LIBTORCH_DIR" => System.get_env("LIBTORCH_DIR", @libtorch_cache),
+        "BUILD_EMBEDDED" =>
+          System.get_env(
+            "BUILD_EMBEDDED",
```

Comment:
Do we need this initial `System.get_env("BUILD_EMBEDDED")`? This is typically set through Mix, not through env vars. :)

Reply:
No, I think we don't need to read it from env. It's easier to change it from a single source.